OpenRA.Game/WorldUtils.cs

```diff
@@ -44,8 +44,8 @@ public static IEnumerable<Actor> FindActorsInCircle(this World world, WPos origi
 
 		public static bool HasVoices(this Actor a)
 		{
-			var selectable = a.Info.Traits.GetOrDefault<SelectableInfo>();
-			return selectable != null && selectable.Voice != null;
+			var voice = a.TraitOrDefault<IVoiced>();
```

Comment:
Let's not assume the actor has only one such trait and keep to
`var voice = a.TraitsImplementing<IVoiced>().FirstOrDefault();`